client-sdk/ts-web/rt/src/types.ts

```diff
@@ -4,10 +4,18 @@ import * as oasis from '@oasisprotocol/client';
  * Arguments for the EstimateGas query.
  */
 export interface CoreEstimateGasQuery {
-    caller?: Uint8Array;
+    caller?: CallerAddress;
     tx: Transaction;
 }
 
+/**
+ * Caller address.
+ */
+export interface CallerAddress {
+    address?: Uint8Array;
+    ethAddress?: Uint8Array;
```

Comment:
In typescript these need to follow serialization as there is no translation afaik?
```suggestion
    eth_address?: Uint8Array;
```